playerUi/separatedAudioSubtitleSettings.html

```diff
@@ -242,6 +245,25 @@ <h2>Your videos play everywhere with low startup delay, no buffering and in high
             hidden: true,
         });
 
+        var subtitleCustomizationSettingsPanel = new bitmovin.playerui.subtitlesettings.SubtitleSettingsPanel({
+            hidden: true,
+            overlay: subtitleOverlay,
+            settingsPanel: subtitleSettingsPanel,
+        });
+
+        var subtitleCustomizationOpenButton = new bitmovin.playerui.subtitlesettings.SubtitleSettingsOpenButton({
+            subtitleSettingsPanel: subtitleCustomizationSettingsPanel,
+            settingsPanel: subtitleSettingsPanel,
+        });
+
+        subtitleSettingsPanel.addComponent(
+            new bitmovin.playerui.SettingsPanelItem(
```

Comment:
Wouldn't it be cleaner to omit the label (which is optional) and add the `SubtitleSettingsOpenButton` in place of the `Spacer`?

Reply:
Yes good point.